app/src/main/java/org/mozilla/fenix/ext/TextView.kt

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.ext
+
+import android.text.style.UnderlineSpan
+import android.widget.TextView
+import androidx.core.text.toSpannable
+
+/**
+ * Adds an underline effect to the text displayed in the TextView.
+ */
+fun TextView.addUnderline() {
+    val currentText = text
+    text = currentText.toSpannable().apply {
+        setSpan(UnderlineSpan(), 0, currentText.length, 0)
+    }
+}
```

Comment:
I think I'd prefer an extension property here, or at least make the method take an optional `start, end` or `start, len` arguments.

Reply:
What sort of property are you thinking of? I'd prefer not to add the arguments as all the uses of UnderlineSpan just underline the entire string.